<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DllViewerClass</class>
 <widget class="QMainWindow" name="DllViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DllViewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../visual-studio-2015-projects/ModuleViewer/ModuleViewer/icons/dll-icon.png</normaloff>../../../visual-studio-2015-projects/ModuleViewer/ModuleViewer/icons/dll-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Type in pid or process name for search:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="searchLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="terminateButton">
          <property name="text">
           <string>Terminate selected process</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startProcessButton">
          <property name="text">
           <string>Start process</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTreeView" name="processView"/>
      <widget class="QTreeView" name="dllsView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1013</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="dllviewer.qrc"/>
 </resources>
 <connections/>
</ui>
